src/viam/sdk/module/handler_map.cpp

```diff
@@ -65,7 +65,7 @@ const HandlerMap_ HandlerMap_::from_proto(viam::module::v1::HandlerMap proto) {
     return hm;
 }
 
-void HandlerMap_::add_model(Model model, RPCSubtype subtype) {
+void HandlerMap_::add_model(const Model& model, const RPCSubtype& subtype) {
```

Comment:
I wonder if it would still want this as a `&` if you `std::move`'d `model` below.